src/components/auth/RouteWithSession.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Route, RouteComponentProps } from "react-router-dom";
+import WithSession from "./WithSession";
+
+export interface RouteWithSessionProps {
+    render: (props: RouteComponentProps<any>) => React.ReactNode;
+    path: string | string[];
```

Comment:
Path is optional for the default path, so `path?: ...`

Reply:
done!